app/Models/User.php

```diff
@@ -59,14 +61,16 @@ class User extends Authenticatable
         'profile_photo_url', 'user_roles'
     ];
 
+
+    use Notifiable;
+
     /**
-     * The webhook for slack notifications.
-     *
-     * @var string
+     * Route notifications for the Slack channel.
      */
-    public function routeNotificationForSlack($notification)
+    public function routeNotificationForSlack(Notification $notification): mixed
     {
-        return env('SLACK_HOOK_NOTIFICATION_URL_NEW_USER_REGISTRATION');
+        //change this to the channel name you want to send notifications to
+        return '#testing';
```

Comment:
This doesn't look right...

Reply:
It is right at the moment later on we can move the function into individual notification channels (the term referring to laravel files eg.RegistrationNotification.php). but until we want our notifications split into different slack channels this is will do (that might deem an issue of its own)